3-cluster-topology.md

```diff
@@ -15,6 +15,100 @@
+
+For reference, this is the declarative block that defines the "`default-worker`" MachineDeployment class in [the ClusterClass spec we originally installed](./yamls/clusterclasses/clusterclass-quick-start.yaml):
+
+```yaml
+  workers:
+    machineDeployments:
+    - class: default-worker
+      template:
+        bootstrap:
+          ref:
+            apiVersion: bootstrap.cluster.x-k8s.io/v1beta1
+            kind: KubeadmConfigTemplate
+            name: quick-start-default-worker-bootstraptemplate
+        infrastructure:
+          ref:
+            apiVersion: infrastructure.cluster.x-k8s.io/v1beta1
+            kind: DockerMachineTemplate
+            name: quick-start-default-worker-machinetemplate
+```
+
+We'll reference the above "`default-worker`" class in [an updated Cluster configuration that declares a second worker pool](yamls/clusters/3-docker-cluster-one-second-worker-pool.yaml).
+
+The difference between this spec and the original spec used to create the "`docker-cluster-one`" Cluster:
+
+```bash
+diff yamls/clusters/1-docker-cluster-one.yaml yamls/clusters/3-docker-cluster-one-second-worker-pool.yaml
+```
+
+Output:
+
+```bash
+35a36,38
+>         - class: default-worker # Adding a 2nd worker pool
+>           name: md-1
+>           replicas: 1
+```
+
+The above shows that we've added a new `MachineDeployment` called `md-1` (Our existing `MachineDeployment` is named `md-0`) to our new spec. By applying this modified spec to our kind management cluster we will initiate the creation of a new node, from this new node pool:
+
+```bash
+kubectl apply -f yamls/clusters/3-docker-cluster-one-second-worker-pool.yaml
+```
+
+Output:
+
+```bash
+cluster.cluster.x-k8s.io/docker-cluster-one configured
+```
+
+Let's watch those new nodes come online:
+
+```bash
+kubectl --kubeconfig cluster-one.kubeconfig get nodes -w
+```
+
+Output:
+
+```bash
+NAME                                             STATUS   ROLES           AGE   VERSION
+docker-cluster-one-md-0-nrh7k-75ddc4778f-wwpg9   Ready    <none>          25h   v1.24.6
+docker-cluster-one-xbqb2-bcjvj                   Ready    control-plane   26h   v1.24.6
+docker-cluster-one-md-1-ksqwt-8489684d5b-fpgjs   NotReady                   <none>          0s    v1.24.6
+docker-cluster-one-md-1-ksqwt-8489684d5b-fpgjs   NotReady                   <none>          46s   v1.24.6
+docker-cluster-one-md-1-ksqwt-8489684d5b-fpgjs   NotReady                   <none>          46s   v1.24.6
+docker-cluster-one-md-1-ksqwt-8489684d5b-fpgjs   Ready                      <none>          76s   v1.24.6
+```
+
+## Remove a worker node
+
+We can now demonstrate how easy it is to "roll back" such a change, as well as show how to remove an existing node pool from your Cluster configuration. In our case it's as easy as reapplying the original Cluster spec, which only declares one "`md-0`" node pool:
```

Comment:
Do we have 3 replicas at this point?

Reply:
At this point there should be two MDs each with one Machine / node

Reply:
And this step removes `md-1`